spreadsheets/kind/boundaries.csv

```diff
@@ -0,0 +1,20 @@
+ne:featurecla,osm:boundary,osm:admin_level,osm:barrier,osm:historic,osm:man_made,min_zoom,kind
+*,administrative,2;4,*,*,*,${8},${NULL}
```

Comment:
We do a [complex transform](https://github.com/mapzen/TileStache/blob/integration-1/TileStache/Goodies/VecTiles/transform.py#L469-L494) in TileStache to get the kind. Created #687 to track.